client/src/components/Reviews/components/AddReview.jsx

```diff
@@ -123,10 +123,10 @@ function AddReview({ handleAddReview, chars }) {
                     <input
                       type="radio"
                       value={val}
-                      checked={val === characteristics[ele]}
+                      checked={val === characteristics[characteristic]}
                       onChange={() => {
                         const newstate = { ...characteristics };
```

Comment:
`setCharacteristic({...characteristics, characteristic: val}) `also works

Reply:
since characteristic is a var would it work? also, its already on the obj so its getting changed not added, I'm not sure familiar with ...

Reply:
oh right in that cause you could do `setCharacteristic({...characteristics, [characteristic]: val}) ` for a dynamic key name. It would just overwrite the property that you already have, like `{a: 1, a:2}` gives `{a:2}`